### ISSUE TITLE
Bot not answering in Skype for Business 

### ISSUE THREAD

**Issue by hucg18** [Microsoft/BotBuilder-V3#138]
## Version
3.15.0

## Describe the bug
Skype for Business bot is replying with an error message:
"Error happened in Skype for Business service when reaching bot service."

Other channels work normally (Webchat and Directline).

## To Reproduce
1.  Open new conversation with the bot in SfB
2. Type anything
3. Get response from bot: "Error happened in Skype for Business service when reaching bot service."

## Expected behavior
Any normal answer from the bot.

## Screenshots
![image](https://user-images.githubusercontent.com/22891658/57119046-eafdc900-6d2c-11e9-89a8-118497641667.png)

## Additional context

[bug]
[javascript]

**Comment by EricDahlvang** [comment]
Hi @hucg18 

I'm sorry you've encountered this issue with the Skype for Business channel.  Have you tried debugging locally? [debug-channel-locally-using-ngrok](https://blog.botframework.com/2017/10/19/debug-channel-locally-using-ngrok/)  

Application insights can also provide useful diagnostic information: 
[Monitor your Node.js services and apps with Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/app/nodejs) 

or 

[Application Insights for ASP.NET Core](https://docs.microsoft.com/en-us/azure/azure-monitor/app/asp-net-core)

Maybe some previous Skype for Business specific issues will provide useful suggestions for how to resolve your issue:
[Unable to get Bot working with Skype for Business](https://github.com/Microsoft/botframework-sdk/issues/5015)

[Bot channel registration stops working with Skype for Business](https://github.com/Microsoft/botframework-sdk/issues/4647)

It is important to note that the Skype for Business channel is being deprecated: 

> Skype for Business channel in Bot Framework is being deprecated on June 30, 2019.
> 
> No new bots will be able to add Skype for Business channel after that date. Existing bots will continue to work until October 31, 2019. Microsoft Teams is the preferred communication tool from Microsoft. Learn how to connect your bot to Microsoft Teams.

From [here](https://docs.microsoft.com/en-us/azure/bot-service/bot-service-channel-connect-skypeforbusiness?view=azure-bot-service-4.0)

**Comment by hucg18** [comment]
Hi Eric, thanks for your answer. 

Indeed we checked the logs in Application Insights and found out what was causing the error:
The cause for this was **debugging the bot locally with ngrok**, if anyone sees this post I strongly recommend to not change the messaging point if they are using Skype for Business as a channel, as I saw in other issues, Skype for business tends to **fail when refreshing the messaging endpoint** thus causing this behavior.

What we finally had to do was delete the 'broken bot' and publish a new one, a real tedious task when you are not the admin of the SfB tenant and depend on a 3rd party.

Thanks for the heads up on the announcement of end of support for SfB bots.
